app/src/main/java/com/github/sdp/mediato/HomeViewModel.java

```diff
@@ -15,20 +14,34 @@
 import java.util.List;
 import java.util.stream.Collectors;
 
-public class HomeViewModel  extends AndroidViewModel {
+/**
+ * The ViewModel for the {@link HomeFragment}
+ */
+public class HomeViewModel extends AndroidViewModel {
```

Comment:
Is there an advantage for using `AndroidViewModel` over just `VIewModel` like in `ProfileViewModel`? It seems like the difference is that it contains the application context but we don't need that here. But I could change `ProfileViewModel` to extend `AndroidViewModel`  in case we need the context in the future and so that we use the same ViewModels.

Reply:
Using View Model does not allow me to get the string from the context. I tested by replacing the AndroidViewModel by ViewModel, and even if the Application is passed, being a ViewModel does not allow the class to get the context.

Reply:
Okay, I will change `ProfileViewModel` then to also use `AndroidViewModel`!